core/ecschema-locaters/src/test/ConcurrentGetSchemaJson.test.ts

```diff
@@ -0,0 +1,135 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Should we have added similar tests for xml files?  How about test cases where some schema files are json and some are xml?

Reply:
Done! Added similar tests for XML files now.